tests/install/openqa_webui.pm

```diff
@@ -72,7 +77,7 @@ git clone https://github.com/os-autoinst/openQA.git
 cd openQA
 for p in $(cpanfile-dump); do echo -n "perl($p) "; done | xargs zypper --non-interactive in -C
 cpanm -nq --installdeps .
-for i in headers proxy proxy_http proxy_wstunnel ; do a2enmod $i ; done
+for i in headers proxy proxy_http proxy_wstunnel rewrite ; do a2enmod $i ; done
```

Comment:
We have the same line in https://github.com/os-autoinst/os-autoinst-distri-openQA/blob/master/tests/install/openqa_webui.pm#L52 . could you extract the common code please?

Reply:
@okurz not sure how to handle this as it is in a <<'EOF'; structure